File: Python/Algorithm/programmers/numPY.py
# [numPY.py]
# 문자열 내 p와 y의 개수

# first
def solution1(s):
    answer = True
    s = s.lower()
    p = 0
    y = 0
    for i in s:
        if i == "p":
            p += 1
        if i == "y":
            y += 1
    if p != y:
        answer = False
    return answer

# second

def solution2(s):
    if s.lower().count("p") == s.lower().count("y"):
        return True
    return False

File: Java/Algorithm/programmers/numPY.java
// [numPY.java]
// 문자열 내 p와 y의 개수

class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int p = 0;
        int y = 0;
        for (int i=0; i<s.length(); i++){
            // p의 개수
            if (s.charAt(i)=='p' || s.charAt(i)=='P'){
                p++;
            // y의 개수
            }else if (s.charAt(i)=='y' || s.charAt(i)=='Y'){
                y++;
            }
        }
        // 같을 경우 true 아닐 경우 false
        return p==y;
    }
}